lambda/src/main/resources/db/migration/V4__transfer_agreement.sql

```diff
@@ -1,11 +1,16 @@
-CREATE TABLE TransferAgreement (
-    ConsignmentId bigint(20) NOT NULL,
-    AllPublicRecords BOOLEAN DEFAULT NULL,
-    AllCrownCopyright BOOLEAN DEFAULT NULL,
-    AllEnglish BOOLEAN DEFAULT NULL,
-    AllDigital BOOLEAN DEFAULT NULL,
-    AppraisalSelectionSignedOff BOOLEAN DEFAULT NULL,
-    SensitivityReviewSignedOff BOOLEAN DEFAULT NULL,
-    PRIMARY KEY (ConsignmentId),
-    CONSTRAINT ConsignmentId FOREIGN KEY (ConsignmentId) REFERENCES Consignment (ConsignmentId)
-)
+CREATE TABLE "TransferAgreement" (
```

Comment:
Should this migration just be included in the new initial migration script?

Reply:
Yeah, you're right, I've moved it.